packages/@aws-cdk/aws-ssm/lib/parameter.ts

```diff
@@ -186,10 +187,25 @@ export enum ParameterType {
 
 export interface StringParameterAttributes {
   /**
-   * The name of the parameter store value
+   * The name of the parameter store value.
+   *
+   * This value can be a token or a concrete string. If it is a concrete string
+   * and includes "/" it must also be prefixed with a "/" (fully-qualified).
    */
   readonly parameterName: string;
 
+  /**
+   * Determines the separator used to render the ARN for the SSM parameter.
+   * Valid values are `"/"` or `""`.
+   *
+   * If `parameterName` is a path (i.e. begins with "/"), the separator must be
+   * `""`. Otherwise, it must be `"/"`.
+   *
+   * @default - automatically determined based on the value of `parameterName`
+   * unless it is a token, in which case this field is required.
+   */
+  readonly parameterArnSeparator?: string;
```

Comment:
Don't entirely understand why we need this. Doesn't a parameter name in an ARN always need to start with a `/`?

Reply:
Ah yeah this looks unused.

Reply:
yeah, old code. nice catch